completions/ip

```diff
@@ -357,7 +357,14 @@ _comp_cmd_ip()
                 all) ;;
                 *)
                     ((cword == subcword)) &&
-                        _comp_compgen -- -W 'help all'
+                        _comp_compgen_split -- "help all $(
+                            {
+                                $1 -c=never monitor help || $1 monitor help
```

Comment:
```suggestion
                                "$1" -c=never monitor help || "$1" monitor help
```

It's better to quote them, though there are still many unquoted cases in the codebase.

Reply:
Fixed